Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -76,6 +82,142 @@ public static class SystemInformation
         public static string DeviceManufacturer { get; }
 
         /// <summary>
+        /// Gets a value indicating whether the app is being used for the first time since it was installed.
+        /// Use this to tell if you should do or display something different for the app's first use.
+        /// </summary>
+        public static bool IsFirstRun { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether the app is being used for the first time since being upgraded from an older version.
+        /// Use this to tell if you should display details about what has changed.
+        /// </summary>
+        public static bool IsAppUpdated { get; }
+
+        /// <summary>
+        /// Gets the first version of the app that was installed.
+        /// This will be the current version if a previous verison of the app was installed before accessing this property.
+        /// </summary>
+        public static string FirstVersionInstalled { get; }
```

Comment:
Should this match the ApplicationVersion return type of PackageVersion?

Reply:
Changed but had to add a PackageVersionHelper as PackageVersion isn't serializable.
Should the methods in this new helper class have extra specific error handling? or happy to just let any formatting errors bubble up from usage?

Reply:
I'm ok with the error bubbling up as you already have the format in the param description